src/components/ha-form/types.ts

```diff
@@ -25,6 +25,8 @@ export interface HaFormBaseSchema {
     suffix?: string;
     // This value will be set initially when form is loaded
     suggested_value?: HaFormData;
+    // Disable flag is passed here from backend for config flow
+    disabled?: boolean;
```

Comment:
This is confusing. We have `disabled` and `description.disabled` and they mean slightly different things. Not clear which has precedence.
I know you are trying to make in the least invasive way but I would prefer a more obvious config. A `readonly` field would be better but still not perfect